### ISSUE TITLE
Randommap implementation

### ISSUE THREAD

**Issue by Alexandroid99** [danielmm8888/TF2Classic#260]
The server lacks the command of the "randommap" command, which would be useful for server operators.

**Comment by LifeDreamer24** [comment]
I agree! The map must be changed manually at each startup otherwise... Wishing to see that in the next update!
